doc/nrf/drivers.rst

```diff
@@ -15,4 +15,4 @@ Here you can find documentation for drivers in the |NCS|, including API document
    :caption: Subpages:
    :glob:
 
-   drivers/*
+   drivers/**
```

Comment:
This flattened the directory structure. So, we might have to go back to your earlier approach, `driver/*` + `drivers/wifi/nrf700x/*`

Reply:
Done